docs/pages/push-notifications/sending-notifications-custom.mdx

```diff
@@ -122,6 +122,18 @@ Your FCM server key can be found by making sure you've followed the [configurati
 
 Communicating with APNs is a little more complicated than with FCM. Some libraries wrap all of this functionality into one or two function calls such as [`node-apn`](https://github.com/node-apn/node-apn). However, in the examples below, a minimum set of libraries are used.
 
+### Client APNs entitlement
+
+The steps below will only work if your iOS app has the APNs entitlement. There are two ways to ensure that this entitlement is added to your app:
+
+- If the app includes the `expo-notifications` package, then the entitlement is added automatically in the Expo configuration.
```

Comment:
This is at odds with what says the issue which this PR is aiming to close, isn't it?

Or maybe I don't understand this.

Reply:
I've rewritten this to make all the requirements clear.